src/store/UI_store/ThemeSlice/ThemeSlice.ts

```diff
@@ -23,7 +23,7 @@ export const ThemeSlice = createSlice({
             initialState.theme = (initialState.theme === 'light') ? 'dark' : 'light'
             initialState.cardBackgroundColor = (initialState.theme === 'light') ? '0F0F0F': '#FFFFFF'
             initialState.backgroundColor = (initialState.theme === 'light') ? '#151515' : '#F8F8FF'
-            initialState.mainTextColor = (initialState.theme === 'light') ? '#EAEAEA': '#333333'
+            initialState.mainTextColor = (initialState.theme === 'light') ? '#333333' : '#EAEAEA'
             initialState.textColor = (initialState.theme === 'light') ? '#7A7A9D' : '#9BABC5'
```

Comment:
I guess that`s actually the same reversed order🙃

Reply:
you are wrong

Reply:
27 line?

Reply:
![image](https://user-images.githubusercontent.com/88248737/229295113-902555dd-ff1d-44fa-9d99-1b983404289c.png)